libkbfs/delayed_cancellation_test.go

```diff
@@ -134,14 +134,14 @@ func TestDelayedCancellationEnabled(t *testing.T) {
 	t.Parallel()
 
 	ctx, cancel := makeContextWithDelayedCancellation(t)
-	EnableDelayedCancellationWithGracePeriod(ctx, 15*time.Millisecond)
+	EnableDelayedCancellationWithGracePeriod(ctx, 50*time.Millisecond)
 
 	cancel()
 
 	select {
 	case <-ctx.Done():
 		t.Fatalf("Cancellation is not delayed")
-	case <-time.After(10 * time.Millisecond):
+	case <-time.After(20 * time.Millisecond):
```

Comment:
Yeah I was concerned about false positive. But I guess if it happens even 1 out of 5, it's still effective. So I'll change this back to 10.